src/main/java/zbot/Storage.java

```diff
@@ -0,0 +1,141 @@
+package zbot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import zbot.task.Deadline;
+import zbot.task.Event;
+import zbot.task.Task;
+import zbot.task.ToDo;
+
+/**
+ * Represents the storage of tasks in a file.
+ */
+public class Storage {
+    private String filePath;
+
+    /**
+     * Constructor for Storage.
+     *
+     * @param filePath Path of the file to store tasks.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Creates the file if it does not exist.
+     *
```

Comment:
A suggestion to add <p> to denote a new paragraph.

If I am not mistaken, this will render the description as 1 line. Therefore possibly try this.

**For example:**
```java
    /**
     * Creates the file if it does not exist.
     * <p>
     * If the file does not exist, the folder containing the file will be created as
     * well.
     * <p>
     * If the file already exists, nothing will be done.
     */
    public void createFileIfNotExists() {
        // Function Body
    }
```

Reply:
I see... did not know that. Thanks!